tests/run/for_in_range_T4550.py

```diff
@@ -0,0 +1,12 @@
+# mode: run
+# tag: forin
+# ticket: t4550
+
+def invalid_arg_count():
```

Comment:
Test looks good to me. I wondering it could be included in https://github.com/cython/cython/blob/master/tests/run/for_in_range_T372.pyx? I think adding a test to a file takes notably less time on the CI than adding a new file.

Reply:
I am fine adding it to another file. I was just not sure whats the preference, since this test file appears to be linked to a specific ticket (which does not exist on github)

Reply:
I added it to https://github.com/cython/cython/blob/master/tests/run/for_in_iter.py, so it tests that the behavior matches in pure python and cython.